### DISCUSSION TITLE
Ask

### DISCUSSION THREAD

**Discussion by BangKuntet** [anasty17/mirror-leech-telegram-bot#801] [Q&A]
Bro Where Is Corect
Use Your Repo Or My Repo
and use brach h-code or master

# Update
UPSTREAM_REPO = "https://https://github.com/anasty17/mirror-leech-telegram-bot"
UPSTREAM_BRANCH = "h-code"

Or

# Update
UPSTREAM_REPO = "https://github.com/(my username github/(my repo)"
UPSTREAM_BRANCH = "master"

**Comment by anasty17** [comment]
1. Heroku ? If yes then use h-code. And it's by default h-code if u r deploying from heroku branch.
2. Better to add ur fork link. Incase u want to update then fetch the new commits then after restart u will get the latest code
3. Branch and link not related to each other like how u doing.
5. Is it hard to read readme ?
